src/watts/plugin_moose.py

```diff
@@ -163,38 +169,37 @@ def prerun(self, params: Parameters):
         """
         self._run_time = time.time_ns()
         # Render the template
-        print("Pre-run for SAM Plugin")
-        super().prerun(params, filename=self.sam_inp_name)
+        print("Pre-run for MOOSE Plugin")
+        super().prerun(params, filename=self.moose_inp_name)
 
     def run(self):
-        """Run SAM"""
-        print("Run for SAM Plugin")
+        """Run MOOSE"""
+        print("Run for MOOSE Plugin")
 
-        log_file = Path("SAM_log.txt")
+        log_file = Path("MOOSE_log.txt")
 
         with log_file.open("w") as outfile:
             func_stdout = tee_stdout if self.show_stdout else redirect_stdout
             func_stderr = tee_stderr if self.show_stderr else redirect_stderr
             with func_stdout(outfile), func_stderr(outfile):
-                run_proc([self.sam_exec, "-i", self.sam_inp_name])
+                run_proc(["mpiexec", "-n", str(self.n_cpu) , self.moose_exec, "-i", self.moose_inp_name])
```

Comment:
Is it safe to assume that MOOSE will always be run with MPI? i.e., is there ever a case where someone doesn't want to use mpiexec?

Reply:
MOOSE requires MPI wrapper compilers during installation, so it should be safe using this approach.